debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution.xml

```diff
@@ -14,12 +15,8 @@
           <scope>runtime</scope>
           <useProjectArtifact>false</useProjectArtifact>
           <excludes>
-              <!-- Exclude all Kafka APIs and their dependencies, since they will be available in the runtime -->
-              <exclude>org.apache.kafka:*</exclude>
-              <exclude>org.slf4j:*</exclude>
+              <!-- Exclude dependencies of Kafka APIs, since they will be available in the runtime -->
```

Comment:
connect-embedded seems to be the only non-test dependency on connect-json which ends up pulling this in. i don't have enough details, but it would be worth considering if this exclusion could be removed now, i.e. only pull in the jackson stuff when it is absolutely needed.

Reply:
_debezium-core_ itself has a dependency to _jackson-core_ as we use it ourselves (and don't want to rely on transitive dependencies). Hence this exclude is still needed.